datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -22,7 +22,7 @@ in-toto==0.4.2
 ipaddress==1.0.22; python_version < '3.0'
 jaydebeapi==1.1.1
 jpype1==0.7.0
-kafka-python==1.4.7; sys_platform != 'win32'
+kafka-python==2.0.0
```

Comment:
Why `sys_platform != 'win32'` was there and not anymore?

Reply:
it works on win, I can remove in separate pr if you want